log.go

```diff
@@ -196,8 +196,18 @@ func (eip *EventInProgress) SetError(err error) {
 }
 
 // Done creates a new Event entry that includes the duration and appended
-// loggables.
-func (eip *EventInProgress) Done() {
+// loggables. Done takes zero or one errors
+func (eip *EventInProgress) Done(err ...error) {
```

Comment:
We could also *add* a `DoneWithErr` function to avoid the weird variadic but I'm mostly ambivalent.

Reply:
This seems like the better option to me, good idea!